ChoiceCoin/Voting
# Teepy SilverBadge

Added an utility module which contains some functions which is going to be important in the course of developing the voting app.

The functions are contained in the `main.py` file of the `choice_coin_voting/utilities` module and also contains a written test cases for every function in `test.py` file.

The list of functionalities are;

1. opt-in any asset into any account.
2. transfer assets from one address to another.
3. checking if an address is a valid algorand address.
4. checks if an algorand key pair is valid.
5. check algorand balance for any account.
6. send algorand from one address to another.
7. check if an account is holding or opted-in an asset.


Testing Instructions

1. install requirements in the `requirements.txt` file (This has been modified to include all project dependencies)
2. cd into the `utilities` folder in the `choice_coin_voting folder`.
3. Modify the content of the `Test` class in the `test.py` file according to the instructions written onto of the file.
4. run `python test.py` and check the results in the terminal.

👍💫👌
